docs/src/pages/guides/markdown-content.md

```diff
@@ -49,65 +49,68 @@ export default {
 };
 ```
 
-### Markdown Pages
+## Markdown Pages
 
-Astro treats any `.md` files inside of the `/src/pages` directory as pages. These pages are processed as plain markdown files and do not support components. If you're looking to embed rich components in your markdown, take a look at the [Markdown Component](#astros-markdown-component) section.
-
-`layout`
-
-The only special Frontmatter key is `layout`, which defines the relative path to an `.astro` component which should wrap your Markdown content.
-
-`src/pages/index.md`
+Astro treats any `.md` files inside of the `/src/pages` directory as pages. These files can contain frontmatter, but are otherwise processed as plain markdown files and do not support components. If you're looking to embed rich components in your markdown, take a look at the [Markdown Component](#astros-markdown-component) section.
 
+### Layouts
+Markdown pages have a special frontmatter property for `layout`. This defines the relative path to an `.astro` component which should wrap your Markdown content, for example a [Layout](/core-concepts/layouts) component. All other frontmatter properties defined in your `.md` page will be exposed to the component as properties of the `content` prop. The rendered Markdown content is placed into the default `<slot />` element.
 ```jsx
+// src/pages/index.md
```

Comment:
I like listing file paths for files, and I think @FredKSchott does too. But would people interpret this as part of the actual code (hopefully not)?

I think I’m in favor of merging this, but whatever pattern we land on we should apply it to all docs pages (ideally in one PR)

Reply:
im in favor of this pattern aswell but i have a question, if we are displaying the route should we display it as `./src/...`